src/caustics/lenses/base.py

```diff
@@ -787,6 +792,7 @@ def surface_density(
         -------
         Tensor
             Surface mass density at the given coordinates in solar masses per Mpc^2.
+        *Unit: solar masses per Mpc^2*
```

Comment:
Make sure to stay consistent. If you are using `solMass` to describe solar mass, then use the same abbreviation throughout.

Reply:
Updated